reference/src/layout/enums.md

```diff
@@ -378,90 +378,31 @@ enum Enum2<T> {
 
 [niche]: ../glossary.html#niche
 
-## Unresolved questions
-
-### Layout of single variant enums
+### Layout of enums with a single variant
 
-[Issue #79.](https://github.com/rust-rfcs/unsafe-code-guidelines/issues/79)
+> **NOTE**: the guarantees in this section have not been approved by an RFC process.
 
-Enums that contain a **single variant** and which do not have an
-explicit `#[repr]` annotation are an important special case. Since
-there is only a single variant, the enum must be instantiated with
-that variant, which means that the enum is in fact equivalent to a
-struct. The question then is to what extent we should **guarantee**
-that the two share an equivalent layout, and also how to define the
-interaction with uninhabited types.
+**Data-carrying** enums with a single variant without a `repr()` annotation have
+the same layout as the variant field. **Fieldless** enums with a single variant
```

Comment:
This is deliberately still use the vague "same layout", right?

Reply:
Right. Making this more precise should be done as part of #153 (I'd like to merge this first, so that we can see the changes in 153 and discuss the naming there).